index.html

```diff
@@ -1,28 +1,44 @@
-<!DOCTYPE HTML >
-<html>
-<head>
-	<title>The network</title>
-	<link rel="stylesheet" type="text/css" href="normalize.css" />
-	<link rel="stylesheet" type="text/css" href="styles.css" />
-</head>
-<body>
-	
-	<div id="header">
-		<div id="logo">
-			<img src="images/logo.png" alt="the network"/>
-		</div>	
-		<div id="search">
-			<form method="get">
-				<input type="text" name="post" placeholder="thoughts?"/>
-				<input type="submit" value="Post">
-			</form>
-		</div>
-		<br>
-	</div>
-	
-	<div id="page">
-
-	</div>
-
-</body>
-</html>
+<!DOCTYPE HTML >
+<html>
+<head>
+	<title>The network</title>
+	<link rel="stylesheet" type="text/css" href="normalize.css" />
+	<link rel="stylesheet" type="text/css" href="styles.css" />
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+	<script>
+		$(document).ready(function(){
+		$('input[value=Post]').on('click',function(event){ /* Check if form was submitted */
+			var $thoughts = $("#search").find('input').val(); /* Find input field data */
+			var dNow = new Date(); /* Set Date Time */
+			var utc = new Date(dNow.getTime() + dNow.getTimezoneOffset() * 60000)
+			var utcdate= (utc.getMonth()+1) + '/' + utc.getDate() + '/' + utc.getFullYear() + ' ' + utc.getHours() + ':' + utc.getMinutes() + ':' + utc.getSeconds();
+			var $day = 	utc.getSeconds();  /* Find seconds */
+			var $answer = ($thoughts+' - '+$day+' <br />');  /* Combine & break data  */
+		$('#page').prepend($answer);  /* Add last input to top of list */
+		return false;  /* Allow the browser to place multiple inputs */
+		document.write($answer);  /* Print to Div */

```

Comment:
Im not sure you need this document.write. You already added the $answer to the DOM on line 17. What are you trying to do here?

Reply:
Yup..meant to remove it. Was writing to div